stactools_landsat/setup.py

```diff
@@ -4,42 +4,44 @@
 from glob import glob
 import io
 
-name = 'stactools_landsat'
-description = ("Subpackage for working with landsat data of stactools, "
-               "a command line tool and Python library for working with STAC.")
+name = "stactools_landsat"
+description = (
+    "Subpackage for working with landsat data of stactools, "
+    "a command line tool and Python library for working with STAC."
+)
 
 __version__ = load_source(
-    'stactools.landsat.version',
-    os.path.join(os.path.dirname(__file__),
-                 'stactools/landsat/version.py')).__version__
+    "stactools.landsat.version",
+    os.path.join(os.path.dirname(__file__), "stactools/landsat/version.py"),
+).__version__
 
-from os.path import (basename, splitext)
+from os.path import basename, splitext
 
 here = os.path.abspath(os.path.dirname(__file__))
 
 # get the dependencies and installs
-with io.open(os.path.join(here, 'requirements.txt'), encoding='utf-8') as f:
-    install_requires = f.read().split('\n')
+with io.open(os.path.join(here, "requirements.txt"), encoding="utf-8") as f:
+    install_requires = f.read().split("\n")
 
 # Add stactools subpackage dependencies
-install_requires.extend(['stactools_core=={}'.format(__version__)])
+install_requires.extend(["stactools_core=={}".format(__version__)])
 
-with open(os.path.join(here, 'README.md')) as readme_file:
+with open(os.path.join(here, "README.md")) as readme_file:
     readme = readme_file.read()
 
-setup(name=name,
-      description=description,
-      version=__version__,
-      long_description=readme,
-      long_description_content_type="text/markdown",
-      author="stac-utils",
-      author_email='stac@radiant.earth',
-      url='https://github.com/stac-utils/stactools.git',
-      packages=find_namespace_packages(),
-      py_modules=[
-          splitext(basename(path))[0] for path in glob('stactools/*.py')
-      ],
-      include_package_data=False,
-      install_requires=install_requires,
-      license="Apache Software License 2.0",
-      keywords=['stactools', 'psytac', 'imagery', 'raster', 'catalog', 'STAC'])
+setup(
+    name=name,
+    description=description,
+    version=__version__,
+    long_description=readme,
+    long_description_content_type="text/markdown",
+    author="stac-utils",
+    author_email="stac@radiant.earth",
+    url="https://github.com/stac-utils/stactools.git",
+    packages=find_namespace_packages(),
+    py_modules=[splitext(basename(path))[0] for path in glob("stactools/*.py")],
+    include_package_data=False,
+    install_requires=install_requires,
+    license="Apache Software License 2.0",
+    keywords=["stactools", "psytac", "imagery", "raster", "catalog", "STAC"],
+)
```

Comment:
Looks like your editor is formatting code in a way that's different than what the project expects - you can use `scripts/format` to format the code with yapf; that will fix the linting stage of the CI. Running `scripts/test` will fail if there's any linting failures from yapf or flake8 (or if the spellcheck or unit tests fail).

Reply:
Ok, got it.